src/fetch.py

```diff
@@ -1,15 +1,25 @@
+import importlib.util
 import json
 import os
+from pathlib import Path
 import requests
 import logging
 import glob
 import urllib3
+import pkgutil
+from types import ModuleType
```

Comment:
Is ModuleType being used somewhere?

Reply:
Not any more! Good catch. Thanks.

Fixed